scripts/darkSubtract.sh

```diff
@@ -37,7 +37,7 @@ if [ "${DARK_FRAME_SUBTRACTION}" = "true" ]; then
 		# Find the closest dark frame temperature wise
 		typeset -i CLOSEST_TEMPERATURE	# don't set yet
 		typeset -i DIFF=100		# any sufficiently high number
-		typeset -i THIS_TEMPERATURE=${THIS_TEMPERATURE}
+		typeset -i THIS_TEMPERATURE=${THIS_TEMPERATURE##0}
```

Comment:
@EricClaeys please check more leading zeros and following zeros

```
pi@allsky:~ $ test=00000700
pi@allsky:~ $ echo ${test##0}
0000700
pi@allsky:~ $ echo ${test##0*0}

pi@allsky:~ $ echo ${test#0*0}
000700
pi@allsky:~ $ echo ${test} | bc
700

```

Reply:
@AndreasLMeg try this:   `echo ${test##*(0)}`

Reply:
pi@allsky:~ $ echo ${test##*(0)}
700

(bashing is so hard for me to understand....)